filter_plugins/filter_hosts.py

```diff
@@ -87,6 +94,30 @@ def get_instance_systemd_service(app_name, inventory_hostname, stateboard):
     return '{}@{}'.format(app_name, inventory_hostname)
 
 
+def get_tasks_dict_from_dir(dir_path):
+    tasks = {}
+
+    if dir_path is None:
+        return tasks
+
+    files = os.listdir(dir_path)
+    files = filter(lambda x: x.endswith('.yml'), files)
+    for file in files:
+        name = os.path.splitext(os.path.basename(file))[0]
+        tasks[name] = os.path.abspath(os.path.join(dir_path, file))
+
+    return tasks
+
+
+def get_tasks_paths(scenario, custom_tasks, custom_tasks_dir):
+    tasks = {
+        **get_tasks_dict_from_dir(ROLE_TASKS_DIR),
+        **get_tasks_dict_from_dir(custom_tasks_dir),
```

Comment:
What if `custom_tasks_dir` doesn't exists?

Reply:
It's `null` by default:
https://github.com/tarantool/ansible-cartridge/blob/d6db741954c91a5fc6840e6d351199c71144895c/defaults/main.yml#L9
And there is a check for a None value in `get_tasks_dict_from_dir`:
https://github.com/tarantool/ansible-cartridge/blob/d6db741954c91a5fc6840e6d351199c71144895c/filter_plugins/filter_hosts.py#L97-L101

Reply:
But what if it's not `None`,  but path is wrong? What about `files = os.listdir('some-bad-non-existent-path')`

Reply:
`files = os.listdir('some-bad-non-existent-path')` will raise `FileNotFoundError`.

But path is checked by config validation:
https://github.com/tarantool/ansible-cartridge/blob/69bd6d60fc474203f1ce215a31cc1f1d791db6a8/library/cartridge_validate_config.py#L430-L435